src/main/java/frc/robot/commands/assist/PrepareForIntake.java

```diff
@@ -0,0 +1,20 @@
+package frc.robot.commands.assist;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.arm.Arm;
+import frc.robot.subsystems.intake.Intake;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+public class PrepareForIntake extends SequentialCommandGroup {
+  // TODO:  read arm angle and intake voltage from GUI
```

Comment:
You should add a comment with the expected behavior.  E.g.:

Prepare For Intake:
1. Sets the Arm to the ideal angle for intake
2. Turns the intake motor on